cmd/mattermost/commands/plugin.go

```diff
@@ -184,3 +220,93 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func pluginPublicKeysCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	pluginPublicKeysResp, appErr := a.GetPluginPublicKeys()
+	if appErr != nil {
+		return errors.New("Unable to list public keys. Error: " + appErr.Error())
+	}
+
+	CommandPrettyPrintln("Listing public keys")
```

Comment:
nit: Wondering if we should omit a header like this. Our CLI output is already "bad", but I'm a fan of the Unix approach to output.

Reply:
Agree, 1. I just went with the pattern already established in the CLI.

Reply:
1/5, but I wouldn't be opposed to breaking that pattern.

Reply:
Works for me.